functions_framework_tool/templates/json/README.md

```diff
@@ -0,0 +1,165 @@
+# JSON example
+
+This example demonstrates writing a function that accepts a JSON request and
```

Comment:
The wording of this title and description threw me off a bit.

We're talking about a HTTP request with JSON (POST body). A JSON request is not a thing.

Suggestions:

- Change the title?
- Don't say "JSON request"?

Reply:
I agree -- wording should be better. This comes directly from `examples/json/README.md` so I'm creating a ticket to fix both at the same time.

Reply:
See #137